sass/px-dropdown.scss

```diff
@@ -72,9 +72,11 @@ $inuit-enable-btn--primary: true;
 .trigger,
 #target {
   width: inherit;
+  flex-grow: var(--px-dropdown-flex-grow, 0);
 }
 .dropdown-trigger {
   width: inherit;
+  min-width: var(--px-dropdown-min-width, inherit);
```

Comment:
@evanjd 
Any reason apply `--px-dropdown-min-width ` as min-width for dropdown-trigger here? According to [the documentation](https://www.predix-ui.com/#/elements/px-dropdown),  it is used to specify min width for the dropdown menu container, right?
![dropdown_document](https://user-images.githubusercontent.com/443663/53080584-f4500580-3533-11e9-8997-ba6135a9c370.png)

When a dropdown is used as iconic menu, obviously it's not applicable to apply `--px-dropdown-min-width` as the min-width for dropdown-trigger since dropdown-trigger is an icon.
<img width="203" alt="iconicmenu" src="https://user-images.githubusercontent.com/443663/53081129-2ca41380-3535-11e9-8434-8861ef7f6682.png">

So could you please define a new variable for dropdown-trigger min with if you need to specify it, or maybe give an option to enable/disable auto apply `--px-dropdown-min-width` to dropdown-trigger?

Thanks.

Reply:
Good catch, I'll raise a PR to fix it now.

Reply:
Raised #102

Reply:
@evanjd  Thanks for the quick response.